packages/core/admin/admin/src/content-manager/components/RelationInput/tests/RelationInput.test.js

```diff
@@ -146,6 +151,20 @@ describe('Content-Manager || RelationInput', () => {
       expect(spy).toHaveBeenCalled();
     });
 
+    test('should call onRelationReorder', () => {
+      const spy = jest.fn();
+      setup({ onRelationReorder: spy });
+
+      const [draggedItem, dropZone] = screen.getAllByLabelText('Drag');
+
+      fireEvent.dragStart(draggedItem);
+      fireEvent.dragEnter(dropZone);
+      fireEvent.dragOver(dropZone);
+      fireEvent.drop(dropZone);
+
+      expect(spy).toHaveBeenCalled();
```

Comment:
Do you think it would make sense to check whether the arguments are also passed properly? I'm not sure if this would test internals too much ...

Reply:
I don't think it's that important for this one, we _could_ do it, but equally we do test the callback is as expected in `RelationInputDataManager`